frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx

```diff
@@ -178,7 +204,7 @@ const CampersTable = ({
                 </Tag>
               );
             })}
-            <ExportButton />
+            <ExportButton campSessionId={campSessionId} campSessionCSVFilename={campSessionCSVFilename}/>
```

Comment:
What are your thoughts on moving the `generateCSV()` function up to the `index.tsx` file? that way we don't have to pass information through, it is the responsibility of the parent function

Reply:
Seems like a better option - I've made the according changes.